npc.rb

```diff
@@ -45,45 +28,43 @@
 options[:repo_regex] = '.*' if options[:repo_regex].nil?
 
 if options[:no_op]
-  puts "RUNNING IN NO-OP MODE"
+  puts 'RUNNING IN NO-OP MODE'
 else
-  puts "MAKING CHANGES TO YOUR REPOS"
+  puts 'MAKING CHANGES TO YOUR REPOS'
 end
 
 util = OctokitUtils.new(options[:oauth])
-repos = util.list_repos(options[:namespace], options)
+parsed = util.load_module_list(options[:file])
+
+parsed.each do |m|
+  next unless options[:merge_conflicts]
+
+  prs = util.fetch_pull_requests("#{m['github_namespace']}/#{m['repo_name']}")
+  prs.each do |pr|
+    # do we already have a label ?
+    pr_merges = util.does_pr_merge("#{m['github_namespace']}/#{m['repo_name']}", pr.number)
+    puts pr_merges
```

Comment:
Perhaps move `puts pr_merges` after the `if pr_merges` on line 48 - we could end up with an output of `nil` otherwise.